jenkins/Jenkinsfile-release

```diff
@@ -45,11 +40,11 @@ node {
         
             sh 'find . -type f -regex ".*/target/symphony-api-client-java-.*.jar" -exec gpg --batch --passphrase $PASSPHRASE -ab {} \\;'
             sh 'ls -la ./symphony-bdk-legacy/symphony-api-client-java/target'
-            sh '''
-                cd ./symphony-bdk-legacy/symphony-api-client-java
-                pwd
-                mvn --batch-mode --settings ../../jenkins/config-release/settings-release.xml -Dgpg.passphrase=$PASSPHRASE -Drepo.id=ossrh -Drepo.username=symphony-platform-solutions -Drepo.password=$PASSPHRASE clean deploy
-            '''
+            sh 'mvn --batch-mode --settings ../../jenkins/config-release/settings-release.xml -P release,$PARAM_REPO_ID ' +
```

Comment:
Idea is to have the following jenkins params:
- PARAM_REPO_ID with default value : symphony-repo. Can be ossrh or symphony-repo
- PARAM_SYM_REPO_URL with default value equals to the symphony repo url. Not used if PARAM_REPO_ID is ossrh